contents/articles/2016-10_ito's_pupil_calibration_and_mapping/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Yuta Itoh's Pupil Calibration and Mapping (Windows)
+date: Mon Oct 17 2016 12:37:11 GMT+0700 (ICT)
+author: Pupil Dev Team
+subtitle: "Yuta Itoh's Pupil calibration and mapping software aims to provide an easy-to-compile C++ implementation of a 3D eye-tracking method..."
+featured_img_thumb: "../../../../media/images/blog/thumb/itoh_pupil.png"
+---
+
+Yuta Itoh's Pupil calibration and mapping software aims to provide an easy-to-compile C++ implementation of a 3D eye-tracking method.
```

Comment:
Yuta Itoh, Jason Orlosky, et al., have implemented and open sourced an easy to compile C++ 3d detection and mapping implementation that runs on Windows!